x-pack/platform/plugins/shared/streams_app/public/components/data_management/stream_detail_enrichment/state_management/simulation_state_machine/simulation_state_machine.ts

```diff
@@ -143,7 +142,11 @@ export const simulationMachine = setup({
     samplingCondition: composeSamplingCondition(input.processors),
     streamName: input.streamName,
   }),
-  initial: 'initializing',
+  initial: 'loadingSamples',
+  invoke: {
+    id: 'subscribeTimeUpdatesActor',
```

Comment:
This subscribes to `timeState$` - the actor emits `dateRange.update` events which are picked up by the machine like before